scripts/netdep_graph_json.py

```diff
@@ -46,6 +46,7 @@ def main():
   with open('clovis-trace.log') as f:
```

Comment:
I'm not 200% confident about how correct my approach is or whether we'll end up needing the clovis graph for something else. Once we're sure clovis is never happening I'll purge all python from lighthouse.